model_card_toolkit/tfx/component.py

```diff
@@ -1,138 +0,0 @@
-"""Model Card TFX Component.
```

Comment:
Sorry for no thinking of this earlier; let's add a deprecation warning mentioning that this component is moving to tfx-addons and link to that PR.